### ISSUE TITLE
Cannot assign a device for operation 'net/Softmax_2'

### ISSUE THREAD

**Issue by Jean-Rassaire** [Kamnitsask/deepmedic#138]
Hi Kamnitsask,

I got this error when trying to train with GPU.

InvalidArgumentError (see above for traceback): Cannot assign a device for operation 'net/Softmax_2': Could not satisfy explicit device specification '/device:GPU:0' because no supported kernel for GPU devices is available.
         [[Node: net/Softmax_2 = Softmax[T=DT_FLOAT, _device="/device:GPU:0"](net/truediv_17)]]

**Comment by Kamnitsask** [comment]
What tensorflow version?

And, did you make any changes in the code?

**Comment by Jean-Rassaire** [comment]
Hi Kamnitsask,

I am using tensorflow 1.5.0

I did not change the code. what I did, i downloaded the python code of the
deepMedicRun and  I removed the if main  then saved as  .py file.

I did that because I am  running on Windows anaconda and did not know how
to run the deepMedicRun file that you provided on anaconda.

it is running  well with CPU, I am getting the error only when I tried  to
run on the GPU.
